### PR TITLE
Addressed Change Analysis UI feedback

### PR DESCRIPTION
**Pull request by rekhaswaminathan**
- Highlight the most recent change group in the default view to indicate that it’s been selected. 
- Change groups should be monotone in color unless selected. 
- Change the cursor to a pointer style when a user hovers over a change group. 
- Make the top row from the change chart expanded by default

### CODE REVIEW

File: AngularApp/projects/diagnostic-data/src/lib/components/changes-view/changes-view.component.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export class ChangesViewComponent extends DataRenderBaseComponent implements OnI
                 });
             });
             this.tableItems.sort((i1, i2) => i1.level - i2.level);
+            this.expandedElement = this.tableItems[0];
```
Comment (yantang-msft) [inline AngularApp/projects/diagnostic-data/src/lib/components/changes-view/changes-view.component.ts]: tableItems can have no elements

File: AngularApp/projects/diagnostic-data/src/lib/components/changesets-view/changesets-view.component.ts [hunk 2]
```diff
@@ -171,6 +170,7 @@ export class ChangesetsViewComponent extends DataRenderBaseComponent implements
         // Create a Timeline
         this.changesTimeline = new Timeline(container, this.timeLineDataSet, this.sourceGroups, options);
         this.changesTimeline.on('select', this.triggerChangeEvent);
+        this.changesTimeline.setSelection(changeSets[0][0]);
```
Comment (yantang-msft) [inline AngularApp/projects/diagnostic-data/src/lib/components/changesets-view/changesets-view.component.ts]: similar that changesets can have no elements

File: AngularApp/projects/diagnostic-data/src/lib/components/changes-view/changes-view.component.ts (hunk 1 above)
Comment (rekhaswaminathan) [reply AngularApp/projects/diagnostic-data/src/lib/components/changes-view/changes-view.component.ts]: this is wrapped in an outer if condition which checks for the rows length.

File: AngularApp/projects/diagnostic-data/src/lib/components/changesets-view/changesets-view.component.ts (hunk 2 above)
Comment (rekhaswaminathan) [reply AngularApp/projects/diagnostic-data/src/lib/components/changesets-view/changesets-view.component.ts]: same here. It is wrapped in an outer if condition which checks if there are rows present.

**Pull request merged**